---a/net/minecraft/item/ItemAppleGold.java
+++b/net/minecraft/item/ItemAppleGold.java
@@ -1,55 +1,36 @@
 package net.minecraft.item;
 
-import java.util.List;
-import net.minecraft.creativetab.CreativeTabs;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.potion.Potion;
 import net.minecraft.potion.PotionEffect;
 import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class ItemAppleGold extends ItemFood
-{
+
+public class ItemAppleGold extends ItemFood {
+
     private static final String __OBFID = "CL_00000037";
 
-    public ItemAppleGold(int amount, float saturation, boolean isWolfFood)
-    {
+    public ItemAppleGold(int amount, float saturation, boolean isWolfFood) {
         super(amount, saturation, isWolfFood);
         this.setHasSubtypes(true);
     }
 
-    @SideOnly(Side.CLIENT)
-    public boolean hasEffect(ItemStack stack)
-    {
-        return stack.getMetadata() > 0;
-    }
-
-    public EnumRarity getRarity(ItemStack stack)
-    {
+    public EnumRarity getRarity(ItemStack stack) {
         return stack.getMetadata() == 0 ? EnumRarity.RARE : EnumRarity.EPIC;
     }
 
-    protected void onFoodEaten(ItemStack stack, World worldIn, EntityPlayer player)
-    {
-        if (!worldIn.isRemote)
-        {
+    protected void onFoodEaten(ItemStack stack, World worldIn, EntityPlayer player) {
+        if (!worldIn.isRemote) {
             player.addPotionEffect(new PotionEffect(Potion.absorption.id, 2400, 0));
         }
 
-        if (stack.getMetadata() > 0)
-        {
-            if (!worldIn.isRemote)
-            {
+        if (stack.getMetadata() > 0) {
+            if (!worldIn.isRemote) {
                 player.addPotionEffect(new PotionEffect(Potion.regeneration.id, 600, 4));
                 player.addPotionEffect(new PotionEffect(Potion.resistance.id, 6000, 0));
                 player.addPotionEffect(new PotionEffect(Potion.fireResistance.id, 6000, 0));
             }
-        }
-        else
-        {
+        } else {
             super.onFoodEaten(stack, worldIn, player);
         }
     }
-
-    @SideOnly(Side.CLIENT)
-    public void getSubItems(Item itemIn, CreativeTabs tab, List subItems)
-    {
-        subItems.add(new ItemStack(itemIn, 1, 0));
-        subItems.add(new ItemStack(itemIn, 1, 1));
-    }
-}
+}
